Formula/d/dependabot-cli.rb

```diff
@@ -0,0 +1,31 @@
+class DependabotCli < Formula
```

Comment:
might be better just call the formula as dependabot rather than dependabot-cli, and consistent with the binary artifact